pymc/tests/test_sampling.py

```diff
@@ -1212,3 +1212,58 @@ def test_sample_deterministic():
         idata = pm.sample(chains=1, draws=50, compute_convergence_checks=False)
 
     np.testing.assert_allclose(idata.posterior["y"], idata.posterior["x"] + 100)
+
+
+class TestDraw(SeededTest):
+    def test_draw_one_variable(self):
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+
+        x_draws = pm.draw(x)
+        assert x_draws["x"].shape == (500,)
+
+    def test_draw_one_variable_with_number_draws(self):
+
+        with pm.Model() as model:
+            x = pm.Normal("x")
+
+        num_draws = 100
+        x_draws = pm.draw(x, draws=num_draws)
+        assert x_draws["x"].shape == (num_draws,)
```

Comment:
This test can be combined with the previous one, just call `pm.draw` twice

Reply:
Hi agree